projects/VitalRecord.Messaging/CommonMessage.cs

```diff
@@ -575,7 +575,7 @@ public static void ValidateMessageHeader(CommonMessage message)
             {
                 throw new MessageRuleException($"Message jurisdiction ID cannot be null.", message);
             }
-            if (message.EventYear == null)
+            if (message.GetYear() == null)
```

Comment:
since there are changes in VitalRecord as well, there needs to be a release of VitalRecord and VitalRecord.Messaging as well